CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+## TBD
+
+* Remove unnecessary gradlew from android directory
```

Comment:
Should note here how this improves the package for users, reducing bundle size etc.